model/registry/file.yaml

```diff
@@ -4,6 +4,41 @@ groups:
     display_name: File Attributes
     brief: "Describes file attributes."
     attributes:
+      - id: file.accessed
+        type: string
+        brief: >
+          Time when the file was last accessed, in ISO 8601 format.
+        note: >
+          This attribute might not be supported by some file systems — NFS, FAT32, in embedded OS, etc.
+        stability: experimental
+        examples: ['2021-01-01T12:00:00Z']
+      - id: file.attributes
+        type: string[]
+        brief: >
+          Array of file attributes.
+        note: >
+          Attributes names depend on the OS or file system. Here’s a non-exhaustive list of values expected for this
+          attribute: `archive`, `compressed`, `directory`, `encrypted`, `execute`, `hidden`, `immutable`, `journaled`, `read`, `readonly`, `symbolic link`, `system`, `temporary`, `write`.
+        stability: experimental
+        examples: ['readonly', 'hidden']
+      - id: file.created
+        type: string
+        brief: >
+          Time when the file was created, in ISO 8601 format.
+        note: >
+          This attribute might not be supported by some file systems — NFS, FAT32, in embedded OS, etc.
+        stability: experimental
+        examples: ['2021-01-01T12:00:00Z']
+      - id: file.ctime
```

Comment:
This can be changed to `file.changed`, everything else looks good to me

Reply:
I was thinking about the same, thanks for double-checking it